app-dicts/myspell-ga/Manifest

```diff
@@ -1 +1 @@
-DIST myspell-ga-20160213.oxt 8475451 BLAKE2B f8fbd0c763df5022142ec824eb7771ee14d0091a9defc5578b5cc830e02d646f031907fd970f69320a8668465c9aa50aa978d05d293fe8c4be24e553052340ec SHA512 bf1d17c8edd174a84c853489a6404da51a21fd47e7abbbd6f1bac98753a6327c6ada7213ffb0fa6bf5ecd62510e8119e35697ae868db00b2086503a801895d0b
+DIST focloiri-gaeilge-5-0.oxt 8748465 BLAKE2B 5fe5147d8f297228bc1e968e60546281110095ab2c3155be2bb31eadd8e486a0f56b7f41bbe661ea568ff191233134955a3a26d271d99d97b3ff82f2f1761e16 SHA512 37c5e03998fd3c036df90f45fa28aba7fbb16fca32df1f82ea3ffeafdb19081b739e6716c5be448f7344b9ec548bbeef09d0bbc4423f6312d53a7a5468c4512c
```

Comment:
The distfile should be identifiable. Is this a version bump too since it goes from 4.8 to 5.0?

Reply:
(If yes, drop this commit, and add a new ebuild with version bump with fixed distfile name)

Reply:
Done